### PR TITLE
Added UI to view and download existing templates

### PR DESCRIPTION
**Pull request by laurence-bird**
UI with ability to look at existing templates and download the raw contents

### CODE REVIEW

File: app/aws/s3/S3Operations.scala [hunk 1]
```diff
@@ -35,21 +37,23 @@ object S3Operations {
...
+    try {
+      templateFiles.foreach {
         case (fileName, file) =>
           zip.putNextEntry(new ZipEntry(fileName))
           zip.write(file)
           zip.closeEntry()
       }
-      Right(baos.toByteArray)
-    } catch{
+      zip.close()
```
Comment (laurence-bird) [inline app/aws/s3/S3Operations.scala]: .close in two places?

File: app/aws/s3/S3Operations.scala [hunk 2]
```diff
@@ -35,21 +37,23 @@ object S3Operations {
...
+      val result = baos.toByteArray
+      Right(result)
+    } catch {
       case e: Throwable => Left(s"Failed to compress template files files: ${e.getMessage}")
     }
-    finally{
+    finally {
       zip.close()
```
Comment (laurence-bird) [inline app/aws/s3/S3Operations.scala]: this can throw an exception

File: conf/application.conf [hunk 3]
```diff
@@ -13,8 +13,8 @@ aws {
   }
   dynamo{
     tables {
-      templateVersionTable = "template-version"
-      templateSummaryTable = "template-summary"
+      templateVersionTable = "template-manager-PRD-TemplateVersionTable-FBZI031JQM9K"
```
Comment (laurence-bird) [inline conf/application.conf]: Change me

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 30](https://circleci.com/gh/ovotech/comms-template-manager/30) at 2017-02-09 11:48 GMT.

**Pull request merged**